src/modules/transfer/views/ViewTransferCreate.vue

```diff
@@ -647,15 +649,31 @@ export default {
     let mosaic = account.assets.find(
       (asset) => asset.idHex == selectedMosaic.value[e.index].id
     );
+    // enable back the option
+    for (let i in mosaics.value){
+      mosaics.value[i].disabled = false
+    }
+    // disable all the options choosen
+    mosaics.value.filter(o1 => selectedMosaic.value.some(o2 => {
+      if(o1.val == o2.id){
+        o1.disabled = true;
+      }
```

Comment:
when using `array.some`, it should return boolean value.
In this case, it is better to use `forEach` instead of `filter`, it do not need to have return value

Reply:
noted, thanks